public/manifest.json

```diff
@@ -1,25 +1,20 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "name": "B.C. Government",
```

Comment:
not sure how important this really is, but I'd recommend using a more descriptive name. Maybe something like "BC Wildfire Predictive Services"?

Reply:
`name` is used for the primary identifier of the app. Yes I will modify `name` and `short_name` to be more descriptive